src/ggrc/converters/base_row.py

```
@@ -145,11 +145,15 @@ def after_save(self, db_session, **options):
       current_user = Person.query.get(current_user_id)
       if current_user and current_user not in self.obj.owners:
         # then create an ObjectOwner connector, add to session
-        db_session.add(ObjectOwner(
-            person_id=current_user_id,
-            ownable=self.obj,
-            modified_by_id=current_user_id
-        ))
+        try:
+          db_session.add(ObjectOwner(
+              person_id=current_user_id,
+              ownable=self.obj,
+              modified_by_id=current_user_id
+          ))
+        except ValidationError as e:
```

@Smotko it's fixed, but I would like to be on a safe side, maybe we should raise an Exception if this happens?